service/src/test/java/bio/terra/workspace/app/controller/ControlledFlexibleResourceApiControllerConnectedTest.java

```diff
@@ -0,0 +1,309 @@
+public class ControlledFlexibleResourceApiControllerConnectedTest extends BaseConnectedTest {
+  private static final Logger logger =
+      LoggerFactory.getLogger(ControlledFlexibleResourceApiControllerConnectedTest.class);
+
+  @Autowired MockMvc mockMvc;
+  @Autowired MockMvcUtils mockMvcUtils;
+  @Autowired ObjectMapper objectMapper;
+  @Autowired UserAccessUtils userAccessUtils;
+  @Autowired JobService jobService;
+  @Autowired GcpCloudUtils cloudUtils;
+  @Autowired FeatureConfiguration features;
+  @Autowired CrlService crlService;
+  @Autowired WorkspaceActivityLogService activityLogService;
+  @Autowired SamService samService;
+
+  private UUID workspaceId;
+  private UUID workspaceId2;
+  private ApiFlexibleResource sourceFlexResource;
+
+  private final String sourceResourceName =
+      TestUtils.appendRandomNumber("source-flexible-resource-name");
+  private static final String sourceTypeNamespace = "terra";
+  private static final String sourceType = "fake-flexible-type";
+  private static final String sourceData = "{\"name\":\"original JSON\"}";
+
+  @BeforeAll
+  public void setup() throws Exception {
+    workspaceId =
+        mockMvcUtils
+            .createWorkspaceWithoutCloudContext(userAccessUtils.defaultUserAuthRequest())
+            .getId();
+    workspaceId2 =
+        mockMvcUtils
+            .createWorkspaceWithCloudContext(userAccessUtils.defaultUserAuthRequest())
+            .getId();
+    // Source flex resource used in clone tests.
+    sourceFlexResource =
+        mockMvcUtils
+            .createFlexibleResource(
+                userAccessUtils.defaultUserAuthRequest(),
+                workspaceId,
+                sourceResourceName,
+                sourceTypeNamespace,
+                sourceType,
+                ControlledFlexibleResource.getEncodedJSONFromString(sourceData))
+            .getFlexibleResource();
+  }
+
+  /**
+   * Reset the {@link FlightDebugInfo} on the {@link JobService} to not interfere with other tests.
+   */
+  @AfterEach
+  public void resetFlightDebugInfo() {
+    jobService.setFlightDebugInfoForTest(null);
+    StairwayTestUtils.enumerateJobsDump(
+        jobService, workspaceId, userAccessUtils.defaultUserAuthRequest());
+    StairwayTestUtils.enumerateJobsDump(
+        jobService, workspaceId2, userAccessUtils.defaultUserAuthRequest());
+  }
+
+  @AfterAll
+  public void cleanup() throws Exception {
+    mockMvcUtils.deleteWorkspace(userAccessUtils.defaultUserAuthRequest(), workspaceId);
+    mockMvcUtils.deleteWorkspace(userAccessUtils.defaultUserAuthRequest(), workspaceId2);
+  }
+
+  // Destination workspace policy is the merge of source workspace policy and pre-clone destination
+  // workspace policy
+  @Test
+  @Disabled(
+      "PF-2563 seems to cause failure upon setup() for controlled api connected tests due to "
+          + "failure with createWorkspaceWithPolicy(...")
+  void clone_policiesMerged() throws Exception {
```

Comment:
I'm having issues running connected controller tests (including GCS bucket and BQ dataset) locally. 

`createWorkspaceWithPolicy` causes an `InitializationError` for me (in all three of these connected controller tests).

I note that the other two connected controller tests are entirely disabled, so maybe this slipped through undetected?